src/python/pants/init/options_initializer.py

```diff
@@ -135,7 +135,6 @@ class OptionsInitializer:
     TODO: We would eventually like to use the bootstrap Scheduler to construct the
     OptionsBootstrapper as well, but for now we do the opposite thing, and the Scheduler is
     used only to resolve plugins.
-      see: https://github.com/pantsbuild/pants/issues/10360
```

Comment:
See https://github.com/pantsbuild/pants/pull/11568 for context, but what this comment was trying to say was that doing this Scheduler->Bootstrapper thing would allow resolving #10360, which we now see as resolved anyway. So...